File: Work/SapORM/DynSapProxy/ProxyObject/DynSapProxyObjectErp35/ErpExtensions.cs
﻿using System.Data;
using System.Globalization;
using ERPConnect;

namespace SapORM.Services
{
    public static class ErpExtensions
    {
        public static DataTable ToADOTableLocaleDe(this RFCTable rfct)
        {
            var tmpTable = rfct.ToADOTable();
            tmpTable.Locale = new CultureInfo("de-DE");
            return tmpTable;
        }
    }
}

File: Work/ZLDBase/Kernel/Common/ErpExtensions.vb
Imports System.Globalization
Imports System.Runtime.CompilerServices
Imports ERPConnect

Module ErpExtensions

    <Extension()> _
    Public Function ToADOTableLocaleDe(rfct As RFCTable) As DataTable
        Dim tmpTable As DataTable = rfct.ToADOTable()
        tmpTable.Locale = New CultureInfo("de-DE")
        Return tmpTable
    End Function

End Module
